pl_bolts/models/autoencoders/basic_ae/basic_ae_module.py

```diff
@@ -122,15 +124,6 @@ def configure_optimizers(self):
     def prepare_data(self):
         self.datamodule.prepare_data()
```

Comment:
why do we still need to call prepare data? shouldn’t that be automatically called for the user?

Reply:
I think @nateraw mentioned here that we need manual calls to prepare_data for now: https://github.com/PyTorchLightning/pytorch-lightning/issues/2742

Reply:
solved here: https://github.com/PyTorchLightning/pytorch-lightning/pull/2755

so we can remove those calls to prepare_data across the repo as soon as this is merged/next release is up on PyPi